aeron-test-support/src/main/java/io/aeron/test/DataCollector.java

```diff
@@ -138,22 +157,98 @@ public Path dumpData(final String destinationDir)
         return copyData(destinationDir);
     }
 
+    /**
+     * Find all the driver cnc files
+     *
+     * @return list of paths to collected driver cnc files.
+     */
     public List<Path> cncFiles()
     {
         return this.locations.stream()
-            .map(p -> p.resolve(CncFileDescriptor.CNC_FILE))
-            .filter(Files::exists)
+            .flatMap(path -> DataCollector.find(path, CncFileDescriptor::isCncFile))
             .collect(toList());
     }
 
-    public List<Path> errorLogFiles()
+    /**
+     * Find all the clustered service mark files.
+     *
+     * @return list of paths to the clustered service mark files.
+     */
+    public List<Path> clusterServiceMarkFiles()
     {
         return this.locations.stream()
-            .map(p -> p.resolve(TestNode.CLUSTER_ERROR_FILE))
-            .filter(Files::exists)
+            .flatMap(path -> DataCollector.find(path, ClusterMarkFile::isServiceMarkFile))
             .collect(toList());
     }
 
+    /**
+     * Find all the consensus module mark files.
+     *
+     * @return list of paths to the consensus module mark files
+     */
+    public List<Path> consensusModuleMarkFiles()
+    {
+        return this.locations.stream()
+            .flatMap(path -> DataCollector.find(path, ClusterMarkFile::isConsensusModuleMarkFile))
+            .collect(toList());
+    }
+
+    /**
+     * Find all the archive mark files.
+     *
+     * @return list of paths to the archive mark files
+     */
+    public List<Path> archiveMarkFiles()
+    {
+        return this.locations.stream()
+            .flatMap(path -> DataCollector.find(path, ArchiveMarkFile::isArchiveMarkFile))
+            .collect(toList());
+    }
+
+    /**
+     * Gets a collection of all registered locations.
+     *
+     * @return list of all locations.
+     */
+    public Collection<Path> allLocations()
+    {
+        return locations;
+    }
+
+    private static Stream<Path> list(final Path p)
```

Comment:
Seems to be unused.

Reply:
Yes, will come through in a subsequent PR.